examples/spl_composites/com.ibm.streamsx.testing.java/src/com/ibm/streamsx/testing/java/TestCustom.java

```diff
@@ -0,0 +1,85 @@
+package com.ibm.streamsx.testing.java;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.ibm.streams.operator.StreamSchema;
+import com.ibm.streams.operator.Type;
+import com.ibm.streamsx.topology.TStream;
+import com.ibm.streamsx.topology.Topology;
+import com.ibm.streamsx.topology.context.StreamsContextFactory;
+import com.ibm.streamsx.topology.spl.SPL;
+import com.ibm.streamsx.topology.spl.SPLStream;
+import com.ibm.streamsx.topology.tester.Condition;
+import com.ibm.streamsx.topology.tester.Tester;
+
+
+/*
+ * This class hows how one can test a SPL composite operator using JUnit
+ * and the Streams Java Application API.
+ */
+public class TestCustom {
+	
+	/*
+	 * Method to create the topology, submit the application based on 
+	 * the specified context and validate data
+	 */
+	private void doTestMultiplyBy(String context) throws Exception {
+		
+		// Create topology to drvie the test
+		Topology topo = new Topology("TestMultiplyBy");
+				
+		// Add the toolkit under test to the toolkit path
+		SPL.addToolkit(topo, new File("../com.ibm.streamsx.testing.spl"));
+		SPL.addToolkit(topo, new File("../com.ibm.streamsx.testing.app"));
+		
+		// Get SPL Schema for test result
+		StreamSchema schema = Type.Factory.getStreamSchema("tuple<int32 result>");
+		
+		// Set up parameters for the composite operator
+		Map<String, Object> params = new HashMap<String, Object>();
```

Comment:
No need to have types on RHS: `new HashMap<`~String, Object~`>()`

Reply:
I am keeping this... as not doing it generates a warning in Eclipse.  Trying to minimize any wanting.